src/reporter/reporter.types.ts

```diff
@@ -0,0 +1,103 @@
+/*
+   Copyright 2022 Mitch Spano
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+	 https://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+import { PluginInputs } from "../common";
+import { ScannerViolation } from "../sfdxCli";
+import { Context } from "@actions/github/lib/context";
+
+export type GithubCheckRun = {
+  name: string;
+  head_sha: string;
+  status: string; // add
+  conclusion:
+    | "action_required"
+    | "cancelled"
+    | "failure"
+    | "neutral"
+    | "success"
+    | "skipped"
+    | "stale"
+    | "timed_out";
+  output: {
+    title: string;
+    summary: string;
+    annotations: GithubAnnotation[];
+  };
+};
+
+export type GithubAnnotation = {
+  path: string;
+  start_side: string;
+  annotation_level: "notice"; // add,
+  start_line: number;
+  end_line: number;
+  message: string;
+  title: string;
+};
+
+export type GithubComment = {
```

Comment:
Is this used in the `annotations-reporter`? If not, it seems like it belongs in `comments-reporter` - right?